index.js

```diff
@@ -1,15 +1,131 @@
-const mbar = document.getElementById('pop');
-const closem = document.querySelector('.close');
-const mPopup = document.querySelector('.menu-up-container');
+function openNav() {
+  document.getElementById('myNav').style.display = 'block';
+}
 
-mbar.addEventListener('click', () => {
-  mPopup.classList.add('pop-up');
-});
+function closeNav() {
+  document.getElementById('myNav').style.display = 'none';
+}
 
-closem.addEventListener('click', () => {
-  mPopup.classList.remove('pop-up');
+document.querySelector('.toggle').addEventListener('click', openNav);
+document.querySelector('.closebtn').addEventListener('click', closeNav);
+document.querySelectorAll('.pop-nav').forEach((link) => {
+  link.addEventListener('click', () => {
+    document.getElementById('myNav').style.display = 'none';
+  });
 });
 
-mPopup.addEventListener('click', () => {
-  mPopup.classList.remove('pop-up');
+const projectname = document.querySelector('.projectname');
+const workSections = document.querySelector('.project-section');
+const modal = document.querySelector('.modal-example');
+const modalImg = document.querySelector('.modal-img');
+const company = document.querySelector('.company');
+const role = document.querySelector('.role');
+const year = document.querySelector('.year');
+
+const projectsList = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/project1.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end dev',
+    company: 'canopy',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Multi-post stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/project2.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Full Stack Dev',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. ',
+    imageLink: 'images/project3.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end Dev',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    imageLink: 'images/project4.png',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Lead developer',
+    company: 'Uber',
+    year: '2018',
```

Comment:
This section is not correctly styled for all projects.

- [ ] For each item, ensure that the icon (gray dot) is positioned correctly for each project.

Compare the images below.
![one](https://user-images.githubusercontent.com/65501535/163578407-56b0a6cd-2cde-4b39-acca-a604fda1c6db.png)

![two](https://user-images.githubusercontent.com/65501535/163578417-aa34ac61-1551-4cbe-8e6f-45ea7d96006e.png)

![three](https://user-images.githubusercontent.com/65501535/163578430-de4bb1a4-7d43-44c4-a0e8-fe25a88719c5.png)